webapp/channels/src/components/channel_settings_modal/channel_settings_configuration_tab.tsx

```diff
@@ -112,8 +112,8 @@ function ChannelSettingsConfigurationTab({channel, setAreThereUnsavedChanges, sh
     const toggleTextPreview = useCallback(() => setShowBannerTextPreview((show) => !show), []);
 
     const hasUnsavedChanges = useCallback(() => {
-        return updatedChannelBanner.text !== initialBannerInfo?.text ||
-            updatedChannelBanner.background_color !== initialBannerInfo?.background_color ||
+        return (updatedChannelBanner.text?.trim() || '') !== (initialBannerInfo?.text?.trim() || '') ||
```

Comment:
Some comment - save the trimmed value rather than always trimming before every single use.